apps/mark/integration-testing/cypress/support/e2e.js

```diff
@@ -17,17 +17,19 @@ import '@testing-library/cypress/add-commands';
 import './commands';
 
 import { electionSampleDefinition } from '@votingworks/fixtures';
-import { makeElectionManagerCard, makePollWorkerCard } from '@votingworks/test-utils';
 import { CardData } from '@votingworks/types';
 import { methodUrl } from '@votingworks/grout';
 
-const ELECTION_MANAGER_CARD_DATA = makeElectionManagerCard(
-  electionSampleDefinition.electionHash,
-  '000000'
-);
-const POLL_WORKER_CARD_DATA = makePollWorkerCard(
-  electionSampleDefinition.electionHash
-);
+const ELECTION_MANAGER_CARD_DATA = {
+  t: 'election_manager',
+  h: electionSampleDefinition.electionHash,
+  p: '000000',
+};
```

Comment:
Will likely restore some form of abstraction here once I've finished Java Card implementation